tests/unit/test_items.py

```diff
@@ -0,0 +1,37 @@
+# coding: utf-8
+
+from __future__ import unicode_literals, absolute_import
+
+import unittest
+import os
+from fmf.base import Tree
+
+# Prepare path to examples
+PATH = os.path.dirname(os.path.realpath(__file__))
+EXAMPLES = PATH + "/../../examples/"
+
+
+class TestGetItems(unittest.TestCase):
+    """ Verify getter of items """
+
+    def setUp(self):
+        self.wget_path = EXAMPLES + "wget"
+        self.wget = Tree(self.wget_path)
+
+
+    def test_item_child(self):
+        item = self.wget['/recursion']['/deep']
+        self.assertEqual(item.get("depth"), 1000)
```

Comment:
Could you assert also 'name' ?
`assert self.wget['/recursion']['/deep'].name == '/recursion/deep' `